.circleci/config.yml

```diff
@@ -15,24 +15,21 @@ commands:
             sudo gem uninstall bundler
             sudo rm /usr/local/bin/bundle || true
             sudo rm /usr/local/bin/bundler || true
-            sudo gem install bundler --version 2.2.21
-            bundle config set path 'vendor/bundle'
```

Comment:
Is removing this `bundle config` a desired change?

Reply:
the configuration has been moved to the `install_dependencies` step

```
bundle install --jobs=3 --retry=3 --path vendor/bundle
```

Just for consistency with [3scale_toolbox confg](https://github.com/3scale/3scale_toolbox/blob/main/.circleci/config.yml#L36-L42). 

I do not have a strong preference on any approach.